config/prod-config.yaml

```diff
@@ -13,9 +13,10 @@ interfaces:
     PORT: 7861
     EXTERNAL_PORT: 7681
     HOST: "0.0.0.0" # either "0.0.0.0" (for public) or "127.0.0.1" (for internal)
-    HOSTNAME: "ppc.mit.edu"  # careful, this is used for the chat service
+    HOSTNAME: "t3desk019.mit.edu"  # careful, this is used for the chat service
     template_folder: "/root/A2rchi/A2rchi/interfaces/chat_app/templates"
     static_folder: "/root/A2rchi/A2rchi/interfaces/chat_app/static"
+    num_responses_until_feedback: 3 #the number of responses given by A2rchi until she asks for feedback.
   uploader_app:
```

Comment:
We may want to increase the num_responsesd_until_feedback to like 5 or 10 so that users don’t get annoyed. Don’t know what the best value is here? People’s thoughts?

Reply:
I think 3 is fine for now, we can increase it if we get negative feedback